sdk/monitor/azure-monitor-opentelemetry-exporter/tests/quickpulse/test_live_metrics.py

```diff
@@ -118,6 +121,7 @@ def test_init(self, generator_mock):
         self.assertEqual(qpm._reader._base_monitoring_data_point, qpm._base_monitoring_data_point)
         self.assertTrue(isinstance(qpm._meter_provider, MeterProvider))
         self.assertEqual(qpm._meter_provider._sdk_config.metric_readers, [qpm._reader])
+        self.assertEqual(qpm._meter_provider._sdk_config.resource, resource)
         self.assertTrue(isinstance(qpm._meter, Meter))
         self.assertEqual(qpm._meter.name, "azure_monitor_live_metrics")
         self.assertTrue(isinstance(qpm._request_duration, Histogram))
```

Comment:
Have you tested with the distro's credential param? Could you add a simple distro test to confirm that the credential follows all the way through to the quickpulse components?

Reply:
Can add this once the exporter code is released.